components/CurrentLocation.js

```diff
@@ -49,6 +55,7 @@ function CurrentLocation () {
 
                 setcurrentBuilding(coord.gn.name);
```

Comment:
i dont see a `setcurrentBuilding` in the hooks?

Reply:
Its there !